lib/adapter/mocha.js

```diff
@@ -58,6 +71,10 @@ function MochaReporter(runner, opts) {
     const status = failures === 0 ? TRConstants.PASSED : TRConstants.FAILED;
     console.log(chalk.bold(status), `${passes} passed, ${failures} failed`);
     client.updateRunStatus(status);
+
+    if (tmpDirPath) {
```

Comment:
Here we do 

```
stopStoringArtifactsInFiles()
```

THis method checks if a temp dir was created. If not - do nothing.